docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -183,6 +183,8 @@ paths:
       responses:
         '201':
           description: Created
+          schema:
+            $ref: '#/definitions/EventType'
```

Comment:
Should add the same for `/event-types/{name} put` then?

Reply:
@a1exsh  Definitely!